### DISCUSSION TITLE
EJBCA 8 is not running on wildfly 31 version

### DISCUSSION THREAD

**Discussion by KiruthigaKadhirvel** [Keyfactor/ejbca-ce#569] [General]
We tried to run EJBCA 8 on Wildfly 31 version, Since EJBCA 8 is not completely migrated to jakarta we are facing compatibility issues with javax listener and filer classes EJBCA 8.

Solution requested:
Compatible EJBCA version with latest Wildfly versions.

**Comment by primetomas** [comment]
I think you will find several questions about this under discussions.
As you say a, upgrade to the latest Jakarta EE is needed for this, and this upgrade (for the first time in Java EE history) is not backwards compatible. So upgrading this will break running on older application servers, and JBoss is not released yet with the latest Jakarta EE support. EJBCA is used a lot in critical deployments so JBoss support is important.

Having said that, you can expect EJBCA being updated for this in the fall, and a new release of JBoss being out sometimes then hopefully as well.

Until then WildFly 26 is the latest supported, all due to Jakarta EE upgrade being not backwards compatible.

**Reply by svenska-primekey** [reply]
Yes, I'm looking forward to this too! We will get there in the near future and we appreciate the communities patience.
